scripts/zod-config-prompt-registry.ts

```diff
@@ -0,0 +1,44 @@
+import { join, dirname } from 'node:path';
+import { createLogger } from '@sap-cloud-sdk/util';
+import { readFile } from 'fs/promises';
+import { existsSync } from 'fs';
+import { generateAndWriteConfig } from './generate-zod-config.js';
+
+const logger = createLogger('generate-zod-config');
+
+// Recursively finds all imported files
+async function findImportedFiles(filePath: string, collectedFiles = new Set<string>()): Promise<void> {
+    console.log(filePath);
```

Comment:
[req] I am thinking maybe keep this as a debug log would be helpful.

Reply:
Was a leftover actually. Removed it.